docs/README.rst

```diff
@@ -975,6 +1002,28 @@ Hooks can be uninstalled by removing them from the list:
   >>> I.__adapt__(0)
 
 
+It is possible to replace or customize the ``__adapt___``
+functionality for particular interfaces.
+
+.. doctest::
+
+   >>> class ICustomAdapt(zope.interface.Interface):
+   ...     @zope.interface.interfacemethod
+   ...     def __adapt__(self, obj):
+   ...          if isinstance(obj, str):
+   ...              return obj
+   ...          return super(type(ICustomAdapt), self).__adapt__(obj)
+
+   >>> @zope.interface.implementer(ICustomAdapt)
+   ... class CustomAdapt(object):
+   ...    pass
```

Comment:
Thanks.

(Sigh. It's because writing doctests—just like typing at the plain Python REPL—requires counting spaces. I counted to four, but (unlike the REPL) that's not enough here because there's the space between the prompt and the beginning of the text that has to be typed and skipped. If anybody has a good Emacs configuration to make writing doctests less eye-gougingly painful, I'd love to hear about it!)